scripts/run_translate_schemas.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+current_dir=$(dirname "${BASH_SOURCE[0]}")
+parent_dir=$(dirname "${current_dir}")
+path_to_parent="$( cd "${parent_dir}" && pwd )"
+
+# Clone translation schema
+git clone -b 885-remove-cy-suffix https://github.com/ONSdigital/eq-translations.git "${path_to_parent}"/temp/eq-translations
+# Install prerequisites and run translation
+(
+  source "$(which virtualenvwrapper.sh)"
+  mkvirtualenv --python="$(which python3)" eq-translations
+  workon eq-translations
+  pip install -r "${path_to_parent}"/temp/eq-translations/requirements.txt
+  cd "${path_to_parent}"/temp/eq-translations || exit
```

Comment:
i would put the `pip install` after the `cd`. that way you don't need to specify the path twice.